purenix/src/Nix/Convert.hs

```diff
@@ -130,50 +188,32 @@ checkKeyword w
   | otherwise = pure w
   where
     -- These idents have a special meaning in purenix.
-    purenixIdents = ["modules", "__ffi"]
+    purenixIdents = ["modules", "__tag", "__ffi"]
```

Comment:
I guess now we're also using `__patternX`, `__fail`, and `__patternFail` as special identifiers as well.

Reply:
also `__fieldX`